docs/userdocs/k8s/README.md

```diff
@@ -46,33 +67,37 @@ python3 virtualenv-16.0.0/virtualenv.py venv
 . venv/bin/activate
 ```
 
-#### With Submarine SDK (Recommended)
+#### Install Submarine SDK
 
-- Install SDK from pypi.org
+**Install SDK from pypi.org (recommended)**
 
 Starting from 0.4.0, Submarine provides Python SDK. Please change it to a proper version needed.
 
 ```bash
 pip install submarine-sdk==0.4.0
```

Comment:
pip install pysubmarine==0.4.0

Reply:
@pingsutw @wangdatan which name should we use? "pysubmarine" or "submarine-sdk"?

Reply:
I'm ok with these two names. I prefer "pysubmarine".
just like arrow SDK named "pyarrow"

Reply:
I've upload `apache-submarine` to PyPi
https://pypi.org/project/apache-submarine/